src/ifcblenderexport/blenderbim/bim/operator.py

```diff
@@ -4316,6 +4316,80 @@ def execute(self, context):
         return {"FINISHED"}
 
 
+class AddToPresentationLayer(bpy.types.Operator):
```

Comment:
Renamed to "Assign" instead of "AddTo" to be consistent with the rest of the code.